ci-scripts/test_agent/main.py

```diff
@@ -33,9 +35,18 @@ def test_done_callback(tester_id, workload, verdict, report):
     return
 
 
+def do_token_refresh():
+    """Run in its own thread and refresh token every 30 min."""
+
+    print("Refreshing Firebase Token:", datetime.now())
+    db_client.user = db_client.auth.refresh(db_client.user["refreshToken"])
```

Comment:
move this to a function inside the client. Avoid accessing members of the client here